device/WakeOnLan/controller.js

```diff
@@ -0,0 +1,130 @@
+﻿'use strict';
+
+const BluePromise = require('bluebird');
+const fs = require('fs')
+const os = require('os');
+const net = require('net');
+const dgram = require('dgram');
+
+const PORT_NBR = 9;
+const DEFAULT_IPADDRESS = '255.255.255.255';
+
+let wolMap = "wol.map";
```

Comment:
two comments here

1. use const instead let (you wont change this name)

1. why don't you use a JSON file to parse (much easier to parse) and/or support passing parameters via command line or env variable